diagnostic.md

```diff
@@ -3,36 +3,48 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+faster loading speeds once page loads.
+routes make more sense.
+allows for quicker development--when understood.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+the handlebars templating library. It is handlebars.
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
 ```sh
-# your answer here
+route.
+model.
+component.
 ```
 
 How does a component differ from an Ember template?
 
 ```sh
-# your answer here
+the template is what is generated with ember.
+The component is a particular part of a route.
+
 ```
 
 What is an opinionated framework? Is Ember opinionated? How or how not?
 
 ```sh
-# your answer here
+A framework may be useful but isn''t necessary to completion of something as
+there are other ways to accomplish a common goal it accomplishes.
+Ember is opinionated in that it is still in relatively early development phases
+and yet is widespread in use.
 ```
 
 In Ember what is a route's job (please list three)?
 
 ```sh
-# your answer here
+directs for where to go to and
+has a router to say that direction.
+has a routes handler.
+has a template that can be changed if needed.
```

Comment:
1. Parse information from the url
2. Link the route to a component or template
3. Load the template/UI data via the 'model' hook.
.5/1